spring-cloud-skipper-shell-commands/src/main/java/org/springframework/cloud/skipper/shell/command/SkipperCommands.java

```diff
@@ -265,4 +268,53 @@ public String upload(@ShellOption(help = "the package to be uploaded") String pa
 		PackageMetadata packageMetadata = skipperClient.upload(properties);
 		return "Package uploaded successfully:[" + packageMetadata.getName() + ":" + packageMetadata.getVersion() + "]";
 	}
+
+	@ShellMethod(key = "list", value = "List of releases with the latest revision on state deployed or failed")
+	public Table listReleases(
+			@ShellOption(help = "wildcard expression to search by release name", defaultValue = NULL) String releaseName) {
+		List<Release> releases = this.skipperClient.getReleases(releaseName);
+		TableModel model = new BeanListTableModel<>(releases, getHeadersForReleaseList());
+		TableBuilder tableBuilder = new TableBuilder(model);
+		TableUtils.applyStyle(tableBuilder);
+		return tableBuilder.build();
+	}
+
+	@ShellMethod(key = "history", value = "List the history of releases for the given release name and max revisions")
+	public Table history(
+			@ShellOption(help = "wildcard expression to search by release name") @NotNull String releaseName,
+			@ShellOption(help = "maximum number of revisions to include in the history", defaultValue = NULL) String max) {
+		Collection<Release> releases = null;
+		if (StringUtils.hasText(max)) {
```

Comment:
We should set the default to something large, e.g. in helm it is 256.  Also, require if it is set to be > 0.  Prob also enforce > 0 on the server side as well.